source/docs/romi-robot/programming-romi-in-labview.rst

```diff
@@ -0,0 +1,37 @@
+Programming the Romi
+====================
+
+Writing a LabVIEW program for the Romi is very similar to writing a program for a regular FRC robot. In fact, all the same tools (LabVIEW, Driver Station, SmartDashboard, etc) can be used with the Romi.
```

Comment:
We need to \ |reg| to add the symbol to the first usage of FRC

Reply:
Embarrassed I missed that. This should actually be spelled out if kept so I will likely rephrase to just eliminate the reference. Probably "regular FRC" to "roboRIO based".

Reply:
Just be careful that FRC is used in several other places too so those will need to be changed or symbol added.